dune_client/client.py

```diff
@@ -71,14 +71,15 @@ def _post(self, route: str, params: Any) -> Any:
         )
         return self._handle_response(response)
 
-    def execute(self, query: Query) -> ExecutionResponse:
+    def execute(self, query: Query, performance: str = "medium") -> ExecutionResponse:
         """Post's to Dune API for execute `query`"""
         response_json = self._post(
             route=f"query/{query.query_id}/execute",
             params={
                 "query_parameters": {
                     p.key: p.to_dict()["value"] for p in query.parameters()
-                }
+                },
+                "performance": performance,
```

Comment:
If we put performance as an attribute of the client, then this would be one of the very few lines of code that needs to change here. Like so,

```suggestion
                "performance": self.performance,
```


The the constructor would be

```py
def __init__(self, api_key: str, performance: str = "medium"):
    self.performance = performance
    ...
```

Reply:
I was thinking about this too, but in our case we initialize the client once and then we want to execute a lot of queries on different cluster sizes.

e.g.
```
dc = DuneClient(...)

dc.refresh(123, 'medium')
dc.refresh(456, 'large')
...
```

Reply:
That's a good idea! So don't merge, ill add some changes

Also, I've found that there is no latest results endpoint implementation. I'm gonna rename this PR and add these changes as well.